tests/tests/reduced-permissions.spec.js

```diff
@@ -0,0 +1,255 @@
+      .contains('Create Cluster Role Binding')
+      .click();
+
+    // subject type - select it first so the list starts loading
+    cy.getIframeBody()
+      .contains('User')
+      .click();
+    cy.getIframeBody()
+      .contains('ServiceAccount')
+      .click();
+
+    // name
+    cy.getIframeBody()
+      .find('[placeholder="Cluster Role Binding Name"]:visible')
+      .type(CRB_NAME);
+
+    // role
+    cy.getIframeBody()
+      .find(
+        '[placeholder="Start typing to select Role Binding from the list."]:visible',
+      )
+      .type(CR_NAME);
+    cy.getIframeBody()
+      .contains(new RegExp(CR_NAME))
+      .click();
+
+    // service account namespace
+    cy.task('getNamespace').then(namespace =>
+      chooseComboboxOption(
+        '[placeholder="Select Namespace"]:visible',
+        namespace,
+      ),
+    );
+
+    // service account name
+    chooseComboboxOption('[placeholder="Select name"]:visible', SA_NAME);
+
+    cy.getIframeBody()
+      .find('[role="dialog"]')
+      .contains('button', 'Create')
+      .click();
+  });
+
+  it('Download kubeconfig for Service Account', () => {
+    // once again the navigation is broken, so clicking on anything with bring us to Cluster Overview
+    cy.reload();
+
+    cy.getLeftNav()
+      .contains('Namespaces')
+      .click();
+    cy.goToNamespaceDetails();
+
+    cy.getLeftNav()
+      .contains('Configuration')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Service Accounts')
+      .click();
+
+    cy.getIframeBody()
+      .contains(SA_NAME)
+      .click();
+
+    cy.getIframeBody()
+      .find('[aria-label="Download Kubeconfig"]')
+      .click();
+
+    cy.task('listDownloads', Cypress.config('downloadsFolder')).then(
+      fileNames => {
+        const kubeconfigFileName = fileNames.find(name =>
+          name.includes(SA_NAME),
+        );
+        tempKubeconfigPath =
+          Cypress.config('downloadsFolder') + '/' + kubeconfigFileName;
+
+        cy.readFile(tempKubeconfigPath).then(e =>
+          cy.writeFile('fixtures/sa-kubeconfig.yaml', e),
+        );
+      },
+    );
+
+    cy.get('[data-testid="app-switcher"]').click();
+
+    // cy.contains('Clusters Overview').click();
+
+    cy.loginAndSelectCluster('sa-kubeconfig.yaml');
+  });
+
+  it('Inspect reduced permissions view', () => {
+    cy.getLeftNav()
+      .contains('Configuration')
+      .should('not.exist');
+
+    // once again the navigation is broken, so clicking on anything with bring us to Cluster Overview
+    cy.reload();
+
+    cy.goToNamespaceDetails();
+
+    // try
+    cy.getIframeBody()
+      .contains('Delete')
+      .click();
+
+    cy.get('[data-testid="luigi-modal-confirm"]').click();
+
+    cy.contains('Failed to delete the Namespace').should('be.visible');
+
+    cy.contains('Close').click();
+
+    cy.getLeftNav()
+      .contains('Workloads')
+      .click();
+
+    cy.getLeftNav()
+      .contains('Deployments')
+      .click();
+  });
```

Comment:
We should delete the cluster-wide resources at the end. We don't need hundreds of Cluster roles on our testing cluster